.env

```
@@ -1,4 +1,15 @@
 SITE_URL=https://zhaohongyu.netlify.app
 SITE_AUTHOR=Zhao Hongyu
 SITE_DESCRIPTION=Zhao Hongyu's portfolio site
-EMAIL_ADDRESS=nathan_115210@hotmail.com
\ No newline at end of file
+EMAIL_ADDRESS=nathan_115210@hotmail.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EMAIL_ADDRESS key should go before the SITE_AUTHOR key

(UnorderedKey)